pkg/analysis/passes/packagejson/packagejson.go

```diff
@@ -33,10 +33,13 @@ func run(pass *analysis.Pass) (interface{}, error) {
 		return nil, nil
 	}
 
-	metadataBody := pass.ResultOf[metadata.Analyzer].([]byte)
+	metadataBody, ok := pass.ResultOf[metadata.Analyzer].([]byte)
+	if !ok {
+		return nil, nil
+	}
 
-	var metadata metadata.Metadata
-	if err := json.Unmarshal(metadataBody, &metadata); err != nil {
+	var md metadata.Metadata
```

Comment:
Not related to this pr, but the previous variable name `metadata` collided with the `metadata` package name, which is not ideal